src/lxc/criu.c

```diff
@@ -191,6 +191,8 @@ static void exec_criu(struct criu_opts *opts)
 		return;
 	}
 
+	static_args += 2 * cgroup_num_hierarchies();
```

Comment:
We pass them on the cli as `--cgroup-root /lxc/foo`, so we need two array elements.